src/main/java/org/example/huobi/model/CryptoData.java

```diff
@@ -0,0 +1,39 @@
+package org.example.huobi.model;
+
+import java.util.List;
+
+public record CryptoData(
```

Comment:
records are available from Java 17, use latest Java always. You can generate them automaticlly from JSON examples using CHATGPT